contracts/VMPX.sol

```diff
@@ -30,9 +28,14 @@ contract VMPX is ERC20("VMPX", "VMPX"), ERC20Capped(108_624_000 ether) {
     }
 
     function mint(uint256 power) external {
-        require(power > 0 && power < 196, 'power out of bounds');
+        require(power > 0, 'power has to be positive');
         require(tx.origin == msg.sender, 'only EOAs allowed');
-        _doWork(power);
+        require(totalSupply() + (BATCH * power) <= cap(), "minting would exceed cap");
```

Comment:
Great catch! please include only this line for now and I'll approve right away; I suggest we move the other part to discussion space. thanks!